acinclude/os-deps.m4

```diff
@@ -715,16 +705,14 @@ AC_CACHE_CHECK(for _res.nsaddr_list, ac_cv_have_res_nsaddr_list,
 ])
 
 
-dnl checks the winsock library to use (ws2_32 or wsock32)
+dnl checks the ws2_32 library to use
```

Comment:
Just a wording polish. AFAICT, this code may end up selecting no library, selecting ws2_32 library, or decide to use some other (unknown to this code by name) winsock library from LIBS.

Simple/direct:

```suggestion
dnl checks whether to use the ws2_32 library
```

or, to match our own `AC_MSG_CHECKING([for winsock library])` inside this code:

```suggestion
dnl checks the winsock library to use
```

I do not insist on this change, and do not want to spend more time on this minor issue. If neither suggestion is an improvement in your opinion, please just resolve this change request without a discussion.